src/components/Display.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Display extends Component {
+  constructor() {
+    super();
+    this.state = {
+
+    };
+  }
+
+  render() {
+    const { result } = this.props;
+    return (
+      <div>
+        Display result
+        {result}
+      </div>
+    );
+  }
+}
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
```

Comment:
There should be a default value for the result prop, ![see this ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)